snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/components/SnomedDescriptionApiTest.java

```diff
@@ -190,6 +187,26 @@ public void deleteDescription() {
 		getComponent(branchPath, SnomedComponentType.DESCRIPTION, descriptionId)
 			.statusCode(404);
 	}
+	
+	@Test
+	public void retireDescription() {
+		String descriptionId = createNewDescription(branchPath);
+		
+		Json requestBody = Json.object(
+			"active", false,
+			"commitComment", "Inactivated description"
+		);
+
+		updateComponent(branchPath, SnomedComponentType.DESCRIPTION, descriptionId, requestBody)
+				.statusCode(204);
+		
+		SnomedDescription description = getComponent(branchPath, SnomedComponentType.DESCRIPTION, descriptionId, "acceptabilities()")
+			.statusCode(200)
+			.extract().as(SnomedDescription.class);
+		
+		assertThat(!description.isActive());
+		assertThat(description.getAcceptabilities().isEmpty());
+	}
```

Comment:
This test is passing without any other changes on 7.x. I suspect something on the client side, as the supposedly retired description appears in `SnomedConceptUpdateRequest` with a non-empty acceptability map. 🔍

Reply:
Yes, I told exactly this to Alina a few hours ago 😄

Reply:
As mentioned earlier, those acceptability values are filled based on the values returned by the server.